odml/dtypes.py

```diff
@@ -196,7 +196,7 @@ def datetime_set(value):
 def boolean_get(string):
     if string is None:
         return None
-    if isinstance(string, unicode):
+    if type(string) in (unicode, str):
```

Comment:
You can use `isinstance()` with a tuple as second argument. No need to switch to `type()`

Reply:
Since `type(sthg)` is used for nearly every comparable check in `odml/dtypes.py` i'd rather keep it consistent for now and change all `type` checks to `isinstance` checks in one go.

Reply:
Fair enough